scripts/generate-app-config/generation-objects/ImportAggregator.js

```diff
@@ -0,0 +1,50 @@
+const IdentifierPlaceholder = require('./IdentifierPlaceholder');
+const path = require('path');
+
+/**
+* This class aggregates imports and helps insure things are included more than once.
```

Comment:
Do you mean, "ensures things are not included more than once" ?

Reply:
Yes. things shouldn't be included more than once. :)